src/Facebook/InstantArticles/Transformer/Rules/Rule.php

```diff
@@ -16,24 +16,6 @@ public function matches($context, $node)
 
         $matches_context = $this->matchesContext($context);
         $matches_node = $this->matchesNode($node);
-        if ($matches_context && $matches_node) {
-            $log->debug('context class: '.get_class($context));
-            $log->debug('context matches: '.($matches_context ? 'MATCHES' : 'no match'));
-            $log->debug('node name: <'.$node->nodeName.' />');
-            $log->debug('node matches: '.($matches_node ? 'MATCHES' : 'no match'));
-            $log->debug('rule: '.get_class($this));
-            $log->debug('-------');
-            return true;
-        }
-        if ($node->nodeName === 'img') {
-            $log->debug('context class: '.get_class($context));
-            $log->debug('context matches: '.($matches_context ? 'MATCHES' : 'no match'));
-            $log->debug('node name: <'.$node->nodeName.' />');
-            $log->debug('node: '.$node->ownerDocument->saveXML($node).' />');
-            $log->debug('node matches: '.($matches_node ? 'MATCHES' : 'no match'));
-            $log->debug('rule: '.get_class($this));
-            $log->debug('-------');
-        }
```

Comment:
Hey @diegoquinteiro, please, do not merge 2 things on same pull request.
This should not be lost unless it is addressed in another way.
This logging information is very helpful when trying to debug the transformation of a document.